lib/compilers/nix.ts

```diff
@@ -0,0 +1,85 @@
+import fs from 'node:fs/promises';
```

Comment:
Please put this after the copyright. Can you run `npm run lint` (which ...ought... to catch this? Or did we lose this @junlarsen do you think?)